src/morse-code.js

```diff
@@ -7,7 +18,28 @@
  * 
  * @returns {string}  - String converted to english or morse code
 */
+function morseCode(message, convertTo) {
+  let result = '';
+  let msgUp = [];
+  let words = [];
+  switch (convertTo) {
+    case 'morse':
+      msgUp = [...message.toUpperCase()];
+      result = msgUp.map((item) => ENG_MOR.get(item)).join(' ');
+      break;
 
-function morseCode(message, convertTo) { }
+    case 'english':
+      words = message.split(/\s{3}/);
+      result = words.map((word) => {
+        const oldChar = word.split(' ');
+        const newChar = oldChar.map((c) => MOR_ENG.get(c));
+        return newChar.join('');
+      }).join(' ');
+      break;
+    default:
+      break;
+  }
```

Comment:
You could've used the same as mentioned [here](https://github.com/ravnhq/nodejs-junior-challenge/pull/17/files#r788338093)